.github/workflows/tests-nightly.yml

```
@@ -315,5 +317,43 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_RELEASES_WEBHOOK_URL }}
           JOB_TITLE: "Test release workflow - ${{github.event.client_payload.project}} ${{github.event.client_payload.version}}"
           JOB_KIND: "release tests"
+        run: |
+          python3 ./.github/workflows/notify/notify-slack.py
+
+  send-slack-notification-scheduled-and-manually:
+    if: ${{always() && github.event_name != 'repository_dispatch'}}
+    runs-on: ubuntu-latest
+    needs: [run-tests-in-parallel]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Install dependencies
+        run: |
+              cd .github/workflows
+              npm ci
```

You can use builtin directory indicator here:
```suggestion
      - name: Install dependencies
        working-directory: .github/workflows
        run: npm ci
```

Also: whole job `send-slack-notification-scheduled-and-manually` runs inside the `.github/workflows`, thus maybe you could define `default` for that?
Like this: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-set-the-default-shell-and-working-directory

---

Thanks!